client/index.jsx

```diff
@@ -14,47 +14,97 @@ const UserLegislatorsInfo = require(__dirname + '/src/components/UserLegislators
 const LegislationSearch = require(__dirname + '/src/components/LegislationSearch.jsx');
 const UserLogin = require(__dirname + '/src/components/UserLogin.jsx');
 const UserSignup = require(__dirname + '/src/components/UserSignup.jsx');
-
-// This is pre-download trimmed down version of Legislator data
-var LegislatorData = require(__dirname + '/src/data/LegislatorData.js');
+const UserLogout = require(__dirname + '/src/components/UserLogout.jsx');
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
-    // This is an cache for storing legislator info retrieved from the Sunlight API.
-    // This is design to reduce the number of API call to sunlight for legislator information
+    this.state = {
+      isVerifyingUserSession: true,
+      isUserLoggedIn: false,
+      username: '',
+      userLocation: {
+        lat: undefined,
+        long: undefined
+      }
+    };
+  }
+
+  // Need check with the server to see if user is autheticated
+  componentDidMount() {
+    $.get('login')
+      .done(function(data) {
+        this.setState({        
+          isVerifyingUserSession: false,
+          isUserLoggedIn: true,
+          username: data.username,
+          userLocation: data.location
```

Comment:
Full address is listed under 'location' in our DB schema and lat+long are separate fields, so I kept location=full address and created 'geoLocation' for lat+long